script.js

```diff
@@ -1 +1,27 @@
-// your code
+let center=document.querySelector(".section-center")
+
+window.addEventListener("DOMContentLoaded",()=>{
+    fetch ('https://gist.githubusercontent.com/maratgaip/44060c688fcf5f2b7b3985a6d15fdb1d/raw/e93c3dce0826d08c8c6e779cb5e6d9512c8fdced/restaurant-menu.json')
+
+    .then(response=>response.json())
+    .then(data=>renderData(data))
```

Comment:
.then(renderData) would enough.

Add comments to your code.

Reply:
comments added